micrOS/source/Server.py

```diff
@@ -242,9 +242,10 @@ async def stream(self, callback, task, content_type):
             data_to_send = b''
 
             while self.connected and data_to_send is not None:
-                data_to_send = callback()
+                data_to_send = await callback()
```

Comment:
I think the callback is non async, it is the sync LM callback to be called, right? As smallest sync logic element is the async loop.

Reply:
See my other comment at [micrOS/source/LM_OV2640.py](https://github.com/BxNxM/micrOS/pull/27/files/cfb5a2c9b5845be0141fd81076601611793b5182#diff-7b4caebd2f3891292c23783959e08d4e5bbc0f8df8029e735ac11cdd3abf22f0).